packages/go_router/CHANGELOG.md

```diff
@@ -68,12 +69,12 @@
 
 ## 4.3.0
 
-- Allows `Map<String, dynamic>` maps as `queryParams` of `goNamed`, `replacedName`, `pushNamed` and `namedLocation`. 
+- Allows `Map<String, dynamic>` maps as `queryParams` of `goNamed`, `replacedName`, `pushNamed` and `namedLocation`.
 
 ## 4.2.9
 
-* Updates text theme parameters to avoid deprecation issues.
-* Fixes lint warnings.
+- Updates text theme parameters to avoid deprecation issues.
+- Fixes lint warnings.
```

Comment:
This isn't in accordance with the [changelog style](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style) for this repo